### PR TITLE
feat: dashboard welcome page ui

### PR DESCRIPTION
**Pull request by NikoManiac**
closes #MASK-49

Please **attention** the `packages/theme/src/changes.ts` and  `packages/theme/src/component-changes.ts`, I change those file to align with ui design theme. If there are questions, we can discuss.

And this PR only work for welcome page UI. And business logic will be covered on next PR.

### CODE REVIEW

File: packages/dashboard/src/components/FooterLine/About.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        width: 580,
+        minHeight: 660,
+        lineHeight: 1.75,
+    },
+    header: {
+        height: 300,
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        background: `url(${new URL('./AboutDialogBackground.png', import.meta.url)}) no-repeat center / cover`,
```
Comment (septs) [inline packages/dashboard/src/components/FooterLine/About.tsx:24]: extract this `new URL` as constant.
```suggestion
        background: `url(${ABOUT_DIALOG_BACKGROUND}) no-repeat center / cover`,
```

**Review by septs — DISMISSED** [review]

File: packages/dashboard/src/components/FooterLine/About.tsx (hunk 1 above)
Comment (NikoManiac) [reply packages/dashboard/src/components/FooterLine/About.tsx]: I created `packages/dashboard/src/assets/index.ts` to manage static resource.

File: packages/dashboard/src/components/FooterLine/Version.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { useDashboardI18N } from '../../locales'
+import { Typography } from '@material-ui/core'
+
+export const Version = ({ className }: { className?: string }) => {
+    const t = useDashboardI18N()
+    const version = globalThis.browser?.runtime.getManifest()?.version ?? process.env.TAG_NAME.slice(1)
+
+    return (
+        <Typography className={className} variant="body2" component="span" color="inherit" fontSize={'inherit'}>
```
Comment (nuanyang233) [inline packages/dashboard/src/components/FooterLine/Version.tsx:9]: Why is it necessary to set `fontSize` when ` variant="body2"` is already used here? Did the designer not provide the text style?

File: packages/dashboard/src/components/FooterLine/links.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "MASK_OFFICIAL_WEBSITE": "https://mask.io",
```
Comment (nuanyang233) [inline packages/dashboard/src/components/FooterLine/links.json:2]: Maybe this should be moved to constant package

File: packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
+import { experimentalStyled as styled } from '@material-ui/core/styles'
+import { getMaskColor } from '@masknet/theme'
+import { InfoOutlined } from '@material-ui/icons'
+import { Button, Typography } from '@material-ui/core'
+import { useNavigate } from 'react-router'
+
+const Container = styled('div')`
```
Comment (nuanyang233) [inline packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx:7]: I recommend to use `makeStyles`

File: packages/dashboard/src/components/FooterLine/Version.tsx (hunk 2 above)
Comment (NikoManiac) [reply packages/dashboard/src/components/FooterLine/Version.tsx:9]: It's not necessary. I will remove it.

File: packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        margin-bottom: 0;
+    }
+
+    & > svg {
+        color: ${getMaskColor(theme).secondaryInfoText}};
+    }
+`,
+)
+
+export const ColumnContentLayout: React.FC = ({ children }) => {
```
Comment (nuanyang233) [inline packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx:60]: memo

File: packages/dashboard/src/components/FooterLine/index.tsx [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+            component="a"
+            style={{ cursor: 'pointer' }}
+            color="textPrimary"
+            className={classes.footerLink}>
+            {children}
+        </MuiLink>
+    )
+}
+
+export function FooterLine() {
```
Comment (nuanyang233) [inline packages/dashboard/src/components/FooterLine/index.tsx:89]: memo

File: packages/dashboard/src/components/FooterLine/links.json (hunk 3 above)
Comment (NikoManiac) [reply packages/dashboard/src/components/FooterLine/links.json:2]: Constant package managed constants of web3. I'm not sure should move links to there.

File: packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx (hunk 5 above)
Comment (NikoManiac) [reply packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx:60]: The components of  RegisterFrame, I will continuou refactor in next pr.

File: packages/dashboard/src/components/FooterLine/links.json (hunk 3 above)
Comment (nuanyang233) [reply packages/dashboard/src/components/FooterLine/links.json:2]: /cc @septs @guanbinrui

Comment (septs) [reply packages/dashboard/src/components/FooterLine/links.json:2]: i think no need.

File: packages/dashboard/src/assets/index.ts [hunk 7]
```diff
@@ -0,0 +1,6 @@
+/**
+ * Manage local static resource
+ */
+export const LOCAL_IMAGE = {
```
Comment (septs) [inline packages/dashboard/src/assets/index.ts:4]: this design no can tree shaking.

plz unpack this object, direct expose `ABOUT_DIALOG_BACKGROUND `

File: packages/dashboard/src/components/FooterLine/About.tsx [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+`,
+)
+
+const brands: Record<string, React.ReactNode> = {
+    'https://www.facebook.com/masknetwork': <FacebookIcon />,
+    'https://twitter.com/realmaskbook': <TwitterIcon />,
+    'https://github.com/DimensionDev/Maskbook': <GitHubIcon />,
+    'https://t.me/maskbook_group': <TelegramIcon />,
+    'https://discord.gg/4SVXvj7': <DiscordIcon />,
+}
```
Comment (septs) [inline packages/dashboard/src/components/FooterLine/About.tsx:106]: @yisiliu request provide shortlink service.

```
r2d2.to/facebook
r2d2.to/twitter
r2d2.to/github
r2d2.to/telegram
r2d2.to/discord
```
etc

File: packages/dashboard/src/components/FooterLine/links.json [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+    "MASKBOOK_GITHUB": "https://github.com/DimensionDev/Maskbook",
+    "MASK_EMAIL": "info@dimension.im",
```
Comment (septs) [inline packages/dashboard/src/components/FooterLine/links.json:4]:
```suggestion
    "MASK_EMAIL": "info@mask.io",
```

Parent issue #3529

File: packages/dashboard/src/components/FooterLine/links.json [hunk 10]
```diff
@@ -0,0 +1,9 @@
...
+    "MOBILE_DOWNLOAD_LINK": "https://mask.io/download-links/#mobile",
```
Comment (septs) [inline packages/dashboard/src/components/FooterLine/links.json:5]: @yisiliu i want this page redesign match homepage style.

File: packages/icons/brands/Discord.tsx [hunk 11]
```diff
@@ -0,0 +1,9 @@
+import { createIcon } from '../utils'
+
+export const DiscordIcon = createIcon(
+    'Discord',
+    <>
+        <path d="M395 151s-40-31-88-35l-4 8c43 11 63 26 83 44-35-18-70-35-131-35s-96 17-131 35c20-18 44-35 83-44l-5-8c-49 5-87 35-87 35s-45 65-53 193c46 52 114 52 114 52l15-19c-25-8-52-24-76-51 28 21 71 44 140 44s112-22 140-44c-23 27-51 43-75 51l14 19s69 0 114-52c-8-128-53-193-53-193zM198 309c-17 0-31-16-31-35s14-36 31-36 31 16 31 36-14 35-31 35zm114 0c-17 0-31-16-31-35s14-36 31-36 31 16 31 36-14 35-31 35z" />
```
Comment (septs) [inline packages/icons/brands/Discord.tsx:6]: @Jack-Works we may be need svg to tsx code-generator or webpack-loader.
CC @guanbinrui

File: packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx (hunk 4 above)
Comment (septs) [reply packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx:7]: @nuanyang233 plz re-read the https://github.com/DimensionDev/Maskbook/blob/develop/docs/css-in-js.md, thanks

File: packages/icons/brands/Discord.tsx (hunk 11 above)
Comment (Jack-Works) [reply packages/icons/brands/Discord.tsx:6]: Yes. I prefer codegen

Comment (septs) [reply packages/icons/brands/Discord.tsx:6]: @Jack-Works plz create a issue, and design a solution.

File: packages/dashboard/src/components/FooterLine/About.tsx (hunk 8 above)
Comment (yisiliu) [reply packages/dashboard/src/components/FooterLine/About.tsx:106]: This could be useful.

**Comment by NikoManiac** [comment]
#3256 fix by commit [fix: update storybook component style](https://github.com/DimensionDev/Maskbook/pull/3552/commits/4b8689d8fbaaa7e7352331e73c744f89e34dd48e)

File: packages/icons/brands/Discord.tsx (hunk 11 above)
Comment (Jack-Works) [reply packages/icons/brands/Discord.tsx:6]: #3563

File: packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx (hunk 4 above)
Comment (Jack-Works) [reply packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx:8]: Here is an update about this guide and the current usage is allowed actually (cause it is simple)

**Review by guanbinrui — APPROVED** [review]

**Review by nuanyang233 — APPROVED** [review]

**Pull request merged**